common/src/main/java/com/wynntils/core/managers/Managers.java

```diff
@@ -52,6 +54,17 @@ public final class Managers {
     public static final DiscoveryManager Discovery = new DiscoveryManager(Net, Territory, MinecraftScheduler);
 
     public static void init() {
-        // We don't need to do anything here, but make sure the class is loaded
+        // Register all manager singletons as event listeners
+
+        FieldUtils.getAllFieldsList(Managers.class).stream()
+                .filter(field -> Manager.class.isAssignableFrom(field.getType()))
```

Comment:
we would do `FieldUtils.readField(field, this, true)` in other parts of the mod. I am not sure which is better

Reply:
Wrong place to comment! Talking about `                        WynntilsMod.registerEventListener(field.get(null));` **field.get(null)**

Reply:
FieldUtils is basically a wrapper. Okay. But it technically does field.get(this) instead of field.get(null)

Reply:
But we are reading a static field so it is prudent to send in null to indicate this..?

Reply:
oh right